packages/web/src/components/result/ReactiveList.js

```diff
@@ -210,7 +210,7 @@ class ReactiveList extends Component {
 						isLoading: false,
 					});
 
-					if (nextProps.hits.length < this.props.hits.length) {
+					if (nextProps.hits === this.props.hits) {
```

Comment:
This condition should be inversed, i.e. `nextProps.hits !== this.props.hits` because if the hits have changed only then the `if` block should execute.